enerdata/contracts/tariff.py

```diff
@@ -394,16 +394,29 @@ def __init__(self):
 
 class T31A(T30A):
     """
-    3.1A Tariff
+    3.1A Tariff patched with 3.1ALB
     """
-    def __init__(self):
+    def __init__(self, **kwargs):
```

Comment:
I think that in this case will be safer if we just define the optional parameter `kva`.

Probably will be simpler and safer, just review if `kva` is provided, if it, assign it to this tariff activating the `self.LB` flag.

What do you think @francescpuig7?

Reply:
Done via: 0e7f4403dc010b2bb55b0d8453463077a809301d